bundles/org.pgcase.xobot.dbproc.runtime/src/org/pgcase/xobot/dbproc/runtime/triggers/TriggerDescriptorImpl.java

```diff
@@ -1,13 +1,13 @@
 package org.pgcase.xobot.dbproc.runtime.triggers;
 
-class TriggerDescriptorImpl implements XTriggerDescriptor {
+public class TriggerDescriptorImpl implements XTriggerDescriptor {
```

Comment:
please revert this change and use TriggerBuilder

Reply:
Done